src/main/webapp/js/instructorFeedbackEdit/feedbackPath.js

```diff
@@ -0,0 +1,70 @@
+$(document).ready(function() {
+    FeedbackPath.attachEvents();
+});
+
+var FeedbackPath = {
+    attachEvents: function() {
+        var allDropdownOptions = $('.feedback-path-dropdown-option');
+        FeedbackPath.attachEventsForAllOptions(allDropdownOptions);
```

Comment:
We'll need at least a `this`, I chose `FeedbackPath` for clarify, and because we are not doing OOP